frontend/src/App.css

```diff
@@ -1,24 +1,15 @@
-.add-post {
-  margin: 20px auto;
-  display: block;
+* {
+  padding: 5px;
 }
 
-.App {
-  margin: 20px;
+.side-nav-bar {
+  border: 1px solid black;
```

Comment:
try to use hex values for colors.
I recommend the website [Flat UI Color](https://flatuicolors.com/)
Black is `#000000`

Reply:
changed